takwimu/urls.py

```diff
@@ -6,14 +6,15 @@
 from takwimu import settings
 
 from takwimu.feed import CountryProfileFeed
-from takwimu.views import HomePageView, SupportServicesView
+from takwimu.views import HomePageView, SupportServicesView, TermsOfUse
 
 urlpatterns = static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT) + \
               static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + \
               [
                   url(r'^$', HomePageView.as_view(), name='home'),
                   url(r'^about/support-services', SupportServicesView.as_view(), name='about_support_services'),
                   url(r'^about$', RedirectView.as_view(permanent=False,   url='/about/')),
+                  url(r'^terms-privacy$', TermsOfUse.as_view(), name='terms-privacy'),
```

Comment:
Why the choice of URL? Is there a better one we could use for this?

Reply:
*terms* would work just fine. I'll make necessary changes.